projects/angular/src/layout/nav/_responsive-nav.clarity.scss

```diff
@@ -118,26 +114,16 @@
 
   .header-hamburger-trigger,
   .clr-nav-close {
-    $trigger-span-psuedo-positioning: -1 * variables.$clr_baselineRem_7px;
+    $trigger-span-psuedo-positioning: calc(-1 * tokens.$cds-global-space-5);
```

Comment:
@Jinnie regarding the following comment

> Nice work. Find some questions below.
> 
> I also can't understand where the hamburger icon change comes from. I know it used to be CSS generated, now it's using the "bars" icon, but I don't see this change in this exact PR.

The odd number (7px) is replaced with even number (8px). Later in code is used for `top` and `bottom` for `:before` and `:after` respectivly.

```
    $trigger-span-psuedo-positioning: calc(-1 * tokens.$cds-global-space-5);
```